recipe/meta.yaml

```diff
@@ -45,23 +45,20 @@ test:
   imports:
     - ultralytics
   commands:
-    # coremltools>=7.0 requires pip install as latest conda coremltools==4.1
-    # - pip install thop hub-sdk
-    # - pip check
+    # Install coremltools from source to fix coremltools<=7.1 bug: ModuleNotFoundError: No module named 'imp'
+    - pip install git+https://github.com/apple/coremltools@main thop hub-sdk
+    - pip check
```

Comment:
it would be great if the pip check would be placed before the installations from pip.

While it is understandable that some tests need things outside of conda-forge, pip check should ideally work for end users without additional install commands from `pypi`.